docs/release-notes/index.md

```diff
@@ -5,6 +5,14 @@
 	
 	These features or endpoints (for the API) are not yet final: syntax might change, response could be different, etc. Don't base your (production) code on experimental features.
 
+## Sept 12, 2023 - GraFx Publisher
+
+![rn_icon](/assets/CHILI_publisher_RGB.svg)
+
+### Fixes
+
+- Fixed document preview: Failure caused by preflight checks failing if required variable is empty
```

Comment:
I think the wording could be better. It's kinda difficult to read now. I would suggest:

Fixed failing document preview because of failing preflight checks if a required variable is empty